src/transformers/modeling_utils.py

```diff
@@ -2740,8 +2769,10 @@ def from_pretrained(
             use_safetensors = False
 
         if is_bitsandbytes_available():
+            is_4bit_serializable = version.parse(importlib.metadata.version("bitsandbytes")) >= version.parse("0.41.3")
             is_8bit_serializable = version.parse(importlib.metadata.version("bitsandbytes")) > version.parse("0.37.2")
         else:
+            is_4bit_serializable = False
```

Comment:
It seems `is_4bit_serializable` is not set into the model - can you set it here: https://github.com/poedator/transformers/blob/save4/src/transformers/modeling_utils.py#L3510 ?

Reply:
Or is it somehow silently set somewhere else?

Reply:
[is_4bit_serializable is set just above, in 2772.](https://github.com/huggingface/transformers/pull/26037/files#diff-6b72b98c4c2dcfc6cc606843917733f5d858374fbc22a735ff483bbc0c1e63eaR2772) Or did you mean something else?

Reply:
I meant setting it as an attribute into the model: `model.is_4bit_serializable = is_4bit_serializable`